### PR TITLE
#680 Update gems to fix security issues

### LINKED ISSUE #680: [FIX] Update gems to fix security issues
This list of gems should be updated to fix the security issues:

https://github.com/pivorakmeetup/pivorak-web-app/network/alerts

### PR DESCRIPTION
**Pull request by myacheg**
Resolves [github issue](https://github.com/pivorakmeetup/pivorak-web-app/issues/680)

Bumped versions: 
* `ruby` up to `2.6.0`;
* `rails`  up to `5.2.2`;
* all gems were bumped to its most recent versions.

### Description

I have opened the issue and tried to figure out which gems exactly should be updated. But link took me to the GitHub's 404. It was early morning and I started playing around with the code. I used a newer `ruby` version, and now it is `2.6.0` (was `2.3.1`). I was expecting a lot of deprecation warnings and upgrade issues. I was surprised. No error has arisen up to this step. But it is only according to tests. Errors start to arise once I performed `rails` upgrade.

### Covered changes during upgrade: 

* During `scss` import, links starting with `https:://` should be passed through [`url()`](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#import) method. 

* One deprecation warning from `shoulda_matchers`  gem presents once you start tests. This will be fixed in `v4.0.0` of `shoulda_matchers` (it is already fixed in [`v4.0.0rc1`](https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_model/validate_inclusion_of_matcher.rb#L274)).

### Next steps

I still have some concerns about hidden 'treasures' of the upgrade. But it seems that upgrade that is finished. 
I will appreciate anybody for any helpful comments with information or link s to material materials on this topic.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.3.1
+2.6.0
```
Comment (sveredyuk) [inline .ruby-version]: Ruby 2.6.0 has `Net::HTTP` security bug. We should wait until 2.6.1 release with fix

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ language: ruby
 dist: precise
 
 rvm:
-  - 2.3.1
+  - 2.6.0
```
Comment (sveredyuk) [inline .travis.yml]: This also requires update ruby at production server. We should schedule DevOps task,

**Review by sveredyuk — COMMENTED** [review]
Good job!

File: .ruby-version (hunk 1 above)
Comment (floor114) [reply .ruby-version]: Now we can use ruby 2.6.1. https://www.ruby-lang.org/en/news/2019/01/30/ruby-2-6-1-released

Comment (myacheg) [reply .ruby-version]: yes. For sure! Will update request today at the evening! =)

**Pull request closed without merging**